packages/react/src/components/ImageCard/ImageHotspots.test.jsx

```diff
@@ -377,4 +399,509 @@ describe('ImageHotspots', () => {
+              attributes: [
+                {
+                  dataSourceId: 'temperature',
+                  label: 'Temp',
+                  precision: 2,
+                  thresholds: [
+                    {
+                      comparison: '>',
+                      value: 0,
+                      icon: 'Error filled',
+                      color: 'purple',
+                    },
+                  ],
+                },
+              ],
+            },
+          },
+          {
+            x: 50,
+            y: 60,
+            content: 'Hotspot4',
+            color: 'green',
+          },
+        ]}
+      />
+    );
+    const hotspot = screen.getByTestId('hotspot-50-60');
+    expect(hotspot).toBeVisible();
+  });
+
+  it('should show matching threshold icons and colors', () => {
+    render(
+      <ImageHotspots
+        height={2048}
+        width={1536}
+        src={portrait}
+        alt="portrait-test-image"
+        hotspots={[
+          {
+            x: 35,
+            y: 65,
+            icon: 'User',
+            color: 'purple',
+            content: {
+              title: 'My Device',
+              description: 'Description',
+              values: { deviceid: '73000', temperature: 35.05 },
+              attributes: [
+                {
+                  dataSourceId: 'temperature',
+                  label: 'Temp',
+                  precision: 2,
+                  thresholds: [
+                    {
+                      comparison: '>',
+                      value: 0,
+                      icon: 'Error',
+                      color: 'purple',
+                    },
+                  ],
+                },
+              ],
+            },
+          },
+        ]}
+      />
+    );
+    const hotspot = screen.getByTestId('hotspot-35-65');
+    expect(hotspot).toBeVisible();
+    expect(hotspot).toHaveAttribute('icon', 'Error');
+    const errorIcon = within(hotspot).getByRole('button').firstChild;
+    expect(errorIcon).toBeVisible();
+    expect(errorIcon).toHaveAttribute('fill', 'purple');
+  });
+
+  it('should render custom icons from an array', () => {
+    render(
+      <ImageHotspots
+        height={683}
+        width={512}
+        src={portrait}
+        alt="portrait-test-image"
+        hotspots={[
+          {
+            x: 10,
+            y: 20,
+            content: <span>Hotspot1</span>,
+            icon: 'CustomError',
+            color: 'red',
+            width: 20,
+            height: 20,
+          },
+        ]}
+        icons={[
+          {
+            id: 'CustomError',
+            icon: Error32,
+            text: 'CustomError',
+          },
+        ]}
+      />
+    );
+    const hotspot = screen.getByTestId('hotspot-10-20');
+    expect(hotspot).toBeVisible();
+    expect(hotspot).toHaveAttribute('icon', 'CustomError');
+  });
+
+  it('should return no icon if not found in the array', () => {
+    const originalDEV = global.__DEV__;
```

Comment:
What's this for?

Reply:
To get coverage over [this portion of code](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/next/packages/react/src/components/ImageCard/ImageHotspots.jsx#L539), because the jest.config.js sets the `__DEV__` environment to false.